<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CActivityDialog</class>
 <widget class="QDialog" name="CActivityDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Activity</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DoppelTimer.qrc">
    <normaloff>:/art/checkmark-2.png</normaloff>:/art/checkmark-2.png</iconset>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mHLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="mTableViewTV">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="mAddTB">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mDeleteTB">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mCloneTB">
           <property name="text">
            <string>Clone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mPickTB">
           <property name="text">
            <string>Pick</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Show activities:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mPeriodCOB">
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Today</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 3 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 3 months</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last year</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mFiltersTB">
           <property name="text">
            <string>More...</string>
           </property>
           <property name="icon">
            <iconset resource="DoppelTimer.qrc">
             <normaloff>:/art/arrowRight.png</normaloff>:/art/arrowRight.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,3,1">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current activity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mCurrentALE">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="mDailyWLE">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Daily time units:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mDailyTULE">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Daily worktime:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="mFiltersGB">
       <property name="title">
        <string>Filters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>State:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mInProgressCB">
          <property name="text">
           <string>In progress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mFinishedCB">
          <property name="text">
           <string>Finished</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mPostponedCB">
          <property name="text">
           <string>Postponed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mCanceledCB">
          <property name="text">
           <string>Canceled</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mClearStateTB">
          <property name="text">
           <string>Clear All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Priority:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mLowCB">
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mNormalCB">
          <property name="text">
           <string>Normal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mHighCB">
          <property name="text">
           <string>High</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mUrgentCB">
          <property name="text">
           <string>Urgent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mClearPriorityTB">
          <property name="text">
           <string>Clear All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mEstimationCB">
          <property name="text">
           <string>Estimation</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="DoppelTimer.qrc">:/art/DTimer96.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;b&gt;DoppelTimer&lt;/b&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="mActionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="mActionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="mActionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="mActionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="DoppelTimer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CActivityDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CActivityDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFiltersTB</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFiltersGB</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>806</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>916</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mInProgressCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>917</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFinishedCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>864</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPostponedCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCanceledCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>864</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mLowCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>842</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNormalCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>855</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>919</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mHighCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mUrgentCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>882</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPeriodCOB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CActivityDialog</receiver>
   <slot>updateFilters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mActionCopy</sender>
   <signal>triggered()</signal>
   <receiver>CActivityDialog</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mActionPaste</sender>
   <signal>triggered()</signal>
   <receiver>CActivityDialog</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mActionCut</sender>
   <signal>triggered()</signal>
   <receiver>CActivityDialog</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mActionDelete</sender>
   <signal>triggered()</signal>
   <receiver>CActivityDialog</receiver>
   <slot>deleteSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFilters()</slot>
  <slot>copy()</slot>
  <slot>paste()</slot>
  <slot>deleteSelection()</slot>
  <slot>cut()</slot>
 </slots>
</ui>
